.github/stale.yml

```diff
@@ -1,17 +1,17 @@
 # Number of days of inactivity before an issue becomes stale
 daysUntilStale: 60
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 7
+daysUntilClose: false
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
   - security
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
+  recent activity. It might be closed if no further activity occurs. Thank you
```

Comment:
I don't know since I'm not familiar with github automation. This file (and some others) were autogenerated by `solidus_dev_support` to match the setup of standard solidus extensions.